doc/node-registry-migration-v2.md

```diff
@@ -1,5 +1,15 @@
 # Node Registry Migration
 
+## Set xmtpd cli branch
+
+The migrator has been tested in commit `8d6f9252956aa0d7113d56e1deeca887c0d1c888`.
```

Comment:
the CLI is getting pushed to GHCR.

In general, docs that use `docker run` are easier to lift and put into some other place as they dont require the repo and golang.

We could also "pin" the version of the docker image in this doc.